release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -167,6 +167,21 @@ For more information about this more flexible SR-IOV network deployment during N
 ==== Update to HAProxy 2.6
 With this release, {product-title} updated to HAProxy 2.6.
 
+[id="ocp-4-14-nmstate-ui-console-update"]
+==== NMstate Operator updated in console
+
+With this release, you can now access the NMstate Operator and resources such as the `NodeNetworkState` (NNS), `NodeNetworkConfigurationPolicy` (NNCP), and `NodeNetworkConfigurationEnhancement` (NNCE) from the web console. See xref:../networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc#k8s-nmstate-updating-node-network-config[Updating node network configuration] for more information on NMState resources and how to update them in the console.
+
+In the *Administrator* perspective of the console to access the NMState Operator and resources follow these steps:
+
+* For the NNCP, navigate to *Networking* -> *NodeNetworkConfigurationPolicy*.
+
+* For the NNCE, navigate to *Networking* -> *NodeNetworkConfigurationPolicy*.
+
+* For the NNS, navigate to *Networking* -> *NodeNetworkState*.
```

Comment:
Typically we don't include procedural content in RNs (the only cases I am specifically aware of are workaround steps for known issues and similar). This information seems to be covered by the procedures in the "Updating node network configuration" page linked above. Suggest deleting these lines.

Reply:
make sense for RNs, thanks @jeana-redhat